packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -66,6 +97,20 @@ abstract class ProgressIndicator extends StatefulWidget {
     super.debugFillProperties(properties);
     properties.add(PercentProperty('value', value, showName: false, ifNull: '<indeterminate>'));
   }
+
+  Widget _buildSemanticsWrapper({
+    @required BuildContext context,
+    @required Widget child,
+  }) {
+    return Semantics(
+      container: true,
+      liveRegion: true,
```

Comment:
Also: If you have an indicator going from 0 to 100 - wouldn't this announce every percent step? Do we need a way tot throttle announcements? E.g. only announce every x step?

Reply:
live region is probably not what you want for a progress indicator, as each value change in itself is unlikely to be interesting. However, some overall "loading" indication might be